benchmarl/environments/magent/common.py

```diff
@@ -0,0 +1,126 @@
+#  Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+#  This source code is licensed under the license found in the
+#  LICENSE file in the root directory of this source tree.
+#
+
+from typing import Callable, Dict, List, Optional
+
+from torchrl.data import CompositeSpec
+from torchrl.envs import EnvBase, PettingZooWrapper
+
+from magent2.environments import (
+    adversarial_pursuit_v4,
+    battle_v4,
+    battlefield_v5,
+    combined_arms_v6,
+    gather_v5,
+    tiger_deer_v4
+)
```

Comment:
We might have to move this into `_get_env` and have some error that says to install the lib in case it is not

Reply:
Ok, I did it.